src/cuga/backend/server/main.py

```
@@ -1255,8 +1372,12 @@ async def event_gen():
 
 
 @app.post("/stream")
-async def stream(request: Request):
+async def stream(
+    request: Request,
+    current_user: Optional[UserInfo] = Depends(get_current_user),
```

Ruff (0.15.2)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)